src/components/Cellar/Keg.js

```diff
@@ -81,7 +91,9 @@ export function Keg({ keg }) {
                     {...{ number: kegValue, formatter: moneyString }}
                   />
                 </p>
-                <p>Potential for spoilage: {spoilageRateDisplayValue}%</p>
+                {!wineService.isWineRecipe(item) && (
```

Comment:
This is a good call-out, as this logic is a little brittle. Instead of the suggested alternative, I improved the semantics for keg spoilage to be more clear and robust in https://github.com/jeremyckahn/farmhand/commit/23f61377e76ef2345f21911f91cb00d35a9b4375.